core/web/jsonld/ThingInterface.php

```diff
@@ -0,0 +1,149 @@
+<?php
```

Comment:
This file exists already as well... 
<img width="347" alt="bildschirmfoto 2017-11-19 um 19 32 13" src="https://user-images.githubusercontent.com/3417221/32993959-6323f8b4-cd60-11e7-9cf7-71516b932ce4.png">